vue.config.js

```
@@ -1,33 +1,39 @@
+const pwaConfig = {
+  name: 'Shipyard',
+  themeColor: '#4DBA87',
+  msTileColor: '#000000',
+  appleMobileWebAppCapable: 'yes',
+  appleMobileWebAppStatusBarStyle: 'black',
+  workboxPluginMode: 'GenerateSW',
+  workboxOptions: {
+    navigateFallback: '/index.html',
+    runtimeCaching: [
+      {
+        urlPattern: new RegExp('^https://'),
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)